jenkins-builds/common-SLES12-2016.10

```diff
@@ -17,6 +17,7 @@
  mc-4.8.18.eb
  nano-2.7.0.eb
  numactl-2.0.11.eb
+ pygel-0.3.1-CrayGNU-2016.11-Python-2.7.12.eb
```

Comment:
There's a typo here, `f` is missing: `pygelf-0.3.1-CrayGNU-2016.11-Python-2.7.12.eb`